config/locales/telephony/en.yml

```diff
@@ -10,15 +10,17 @@ en:
         %{app_name}: Your one-time code is %{code}. It expires in %{expiration} minutes. Don't share this code with anyone.
 
         @%{domain} #%{code}
-      voice: Hello! Your %{app_name} one time passcode is, %{code}, again, your
-        passcode is, %{code}, This code expires in %{expiration} minutes.
+      voice: Hello! Your %{app_name} one time passcode is, %{code}. Your one time
+        passcode is, %{code}. Again, your passcode is, %{code}. This code
+        expires in %{expiration} minutes.
```

Comment:
Can you confirm if this is the final content? I had seen some discussion on Slack about changing "passcode" to "code" for consistency. I expect we'd also need translations for these changes.

Reply:
Final content will be "Hello! Your Login.gov one-time code is, 123456. Your one-time code is, 123456. Again, your one-time code is 123456. This code expires in 10 minutes."

I can change this reference here, and throughout the codebase (thankfully, there aren't many references "one-time passcode" to the forward-facing user). I will also check in with UX team so that they can be on the same page.